spec/acceptance/ad_units_spec.rb

```diff
@@ -8,14 +8,15 @@
     {
       title: ad_unit.title,
       description: ad_unit.description,
-      click_url: ad_unit.click_url,
-      ad_unit_url: ad_unit.click_url,
+      click_url_default: ad_unit.click_url_default,
```

Comment:
Add all parameters to these tests, not just the ones that are validated. We use these specs to communicate the API endpoints to front end engineers so we should let them know everything that can be passed.